product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "EDB Postgres Distributed Proxy"
+navTitle: "PGD Proxy"
+---
+
+EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the BDR consensus mechanism to obtain the identity of the current write leader node and redirects traffic to that node.
+
+There is always at least one global group and one data group in the PGD cluster. BDR elects the write leader for each data group which has the `enable_proxy_routing` option set to true. Proxy can be attached to a global group or data group. There could be multiple proxies attached to each group.
+
+PGD Proxy is a TCP layer 4 proxy. It can't encode/decode the PostgreSQL Wire Protocol. Although this may change in future releases.
+
+
+## How it works
+
+Upon starting, PGD Proxy connects to one of the endpoints given in the local config file and fetches DB connection information for all nodes, proxy options like listen address, listen port, routing details like current write leader, etc.
+
+The write leader election and routing is managed by BDR itself. Proxy internally maintains watchers and PubSub mechanism to get leader change notifications on Postgres notify/listen channels.
+
+All application client traffic then passes through the Proxy into the current write leader. If the write leader is not available then, Proxy disconnects all connection traffic until a new write leader is available. This also applies to circumstances when `pgd switchover` command is used to invoke a planned transition to a new write leader. The disconnect is immediate.
```

Comment:
may be need to re-phrase this para

`on write leader change proxy will disconnect existing client conn`

Reply:
`The disconnect is immediate.` ? may be not required